doc/source/ray-air/examples/analyze_tuning_results.ipynb

```diff
@@ -0,0 +1,918 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(train_dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d996bdd",
+   "metadata": {},
+   "source": [
+    "Ray AIR allows us to define and tune preprocessors. For instance, we may want to decide if we should scale the \"Wilderness area\" parameters or the \"Soil type\" parameters. For both, we'll just try standard scalers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "ef7b2992",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's tune the preprocessors\n",
+    "prep_v1 = StandardScaler(columns=[f\"Wilderness_Area_{i}\" for i in range(4)])\n",
+    "prep_v2 = StandardScaler(columns=[f\"Soil_Type_{i}\" for i in range(40)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "282a6078",
+   "metadata": {},
+   "source": [
+    "Since we'll be training a multiclass prediction model, we have to pass some information to XGBoost. For instance, XGBoost expects us to provide the number of classes, and multiclass-enabled evaluation metrices.\n",
+    "\n",
+    "With these parameters in place, we'll create a Ray AIR `XGBoostTrainer`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "bbece53a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# XGBoost specific params\n",
+    "params = {\n",
+    "    \"tree_method\": \"approx\",\n",
+    "    \"objective\": \"multi:softmax\",\n",
+    "    \"eval_metric\": [\"mlogloss\", \"merror\"],\n",
+    "    \"num_class\": 8,\n",
+    "}\n",
+    "\n",
+    "trainer = XGBoostTrainer(\n",
+    "    scaling_config={\"num_workers\": 2},\n",
```

Comment:
Add a comment on what scaling config is?

Until this point, I actually didn't know that each trial was itself going to be distributed training. Might be worth mentioning this earlier.